### PR TITLE
docs(operators): add documentation for windowToggle

### LINKED ISSUE #126: Add docs for operator : transformation - windowToggle
Add docs for operator windowToggle under group transformation 
 
Below is minimum details needed 
 
- [ ] name 
- [ ] operatorType 
- [ ] signature 
- [ ] parameters 
- [ ] short description 
- [ ] Technical description 
- [ ] Visual example  (Marble diagram) 
- [ ] Real world example (Non-framework) 
- [ ] Related Operators

### PR DESCRIPTION
**Pull request by hardikpthv**
Close #126

### CODE REVIEW

File: src/operator-docs/transformation/windowToggle.ts [hunk 1]
```diff
@@ -1,6 +1,81 @@
 import { OperatorDoc } from '../operator.model';
 
 export const windowToggle: OperatorDoc = {
-  'name': 'windowToggle',
-  'operatorType': 'transformation'
+  name: 'windowToggle',
+  operatorType: 'transformation',
+  signature:
+    'public windowToggle(openings: Observable<O>, closingSelector: function(value: O): Observable): Observable<Observable<T>>',
```
Comment (DiedrikDM) [inline src/operator-docs/transformation/windowToggle.ts]: In contrast with the old documentation, we don't want any "generics" in the method signature as per [#196](https://github.com/ReactiveX/rxjs-docs/pull/196#discussion_r155799478). So please remove the generics.

File: src/operator-docs/transformation/windowToggle.ts [hunk 2]
```diff
@@ -1,6 +1,81 @@
...
+      code: `
+      import { mergeAll, windowToggle } from 'rxjs/operators';
+
+      const clicks = Rx.Observable.fromEvent(document, 'click');
+      const openings = Rx.Observable.interval(1000);
+      const result = clicks.windowToggle(openings, i =>
+        i % 2 ? Rx.Observable.interval(500) : Rx.Observable.empty()
+      ).mergeAll();
+      result.subscribe(x => console.log(x));
+    `,
```
Comment (DiedrikDM) [inline src/operator-docs/transformation/windowToggle.ts]: Add some expected console output as a comment below the subscription in the code. Also add it in the jsbin example.

File: src/operator-docs/transformation/windowToggle.ts (hunk 1 above)
Comment (hardikpthv) [reply src/operator-docs/transformation/windowToggle.ts]: Okay, will do it

File: src/operator-docs/transformation/windowToggle.ts (hunk 2 above)
Comment (hardikpthv) [reply src/operator-docs/transformation/windowToggle.ts]: Got it.

File: src/operator-docs/transformation/windowToggle.ts [hunk 3]
```diff
@@ -1,6 +1,102 @@
...
-  'operatorType': 'transformation'
+  name: 'windowToggle',
+  operatorType: 'transformation',
+  signature:
+    'public windowToggle(openings: Observable, closingSelector: function(value: O): Observable): Observable',
+  marbleUrl: 'http://reactivex.io/rxjs/img/windowToggle.png',
+  parameters: [
+    {
+      name: 'openings',
+      type: 'Observable<O>',
```
Comment (JWO719) [inline src/operator-docs/transformation/windowToggle.ts]: maybe remove this generic. Could be confusing

File: src/operator-docs/transformation/windowToggle.ts [hunk 4]
```diff
@@ -1,6 +1,102 @@
...
+      name: 'Every other second, emit the click events from the next 500ms',
+      code: `
+      import { fromEvent } from 'rxjs/observable/fromEvent';
+      import { empty } from 'rxjs/observable/empty';
+      import { interval } from 'rxjs/observable/interval';
+      import { mergeAll, windowToggle } from 'rxjs/operators';
+
+      const clicks = fromEvent(document, 'click');
+      const openings = interval(1000);
+      const result = clicks.windowToggle(openings, i =>
```
Comment (ashwin-sureshkumar) [inline src/operator-docs/transformation/windowToggle.ts]: This should be using `.pipe`

Comment (hardikpthv) [reply src/operator-docs/transformation/windowToggle.ts]: Okay, will do it.

File: src/operator-docs/transformation/windowToggle.ts (hunk 3 above)
Comment (hardikpthv) [reply src/operator-docs/transformation/windowToggle.ts]: hmm, will remove it.

**Pull request merged**